### PR TITLE
feat(server):export role as Prometheus metric

### PR DESCRIPTION
**Pull request by ashotland**
This enable filtering instances by role in dashbaords.

### CODE REVIEW

File: src/server/server_family.h [hunk 1]
```diff
@@ -61,6 +61,7 @@ struct Metrics {
   uint64_t ooo_tx_transaction_cnt = 0;
   uint32_t traverse_ttl_per_sec = 0;
   uint32_t delete_ttl_per_sec = 0;
+  bool is_master = true;
```
Comment (kostasrim) [inline src/server/server_family.h:64]:
```suggestion
  bool is_master = false;
```

File: src/server/server_family.cc [hunk 2]
```diff
@@ -1513,8 +1515,11 @@ Metrics ServerFamily::GetMetrics() const {
   result.traverse_ttl_per_sec /= 6;
   result.delete_ttl_per_sec /= 6;
 
-  if (ServerState::tlocal() && ServerState::tlocal()->is_master)
+  result.is_master = false;
```
Comment (kostasrim) [inline src/server/server_family.cc:1518]:
```suggestion
```

**Review by kostasrim — APPROVED** [review]
I accept this and the comments are optional. 

The reason I am suggesting to default `is_master` to `false` is because you explicitly assume that it's false on line 1518 with the assign,ent `result.is_master = false` which you later set to true within the if statement (if it's actually a master). By doing this change we save 1 line in the body of the function :)

**Comment by ashotland** [comment]
> I accept this and the comments are optional.
> 
> The reason I am suggesting to default `is_master` to `false` is because you explicitly assume that it's false on line 1518 with the assign,ent `result.is_master = false` which you later set to true within the if statement (if it's actually a master). By doing this change we save 1 line in the body of the function :)

I was debating with myself about this too. 
Thought it would be weird to have is_master = false by default as the server is master by default.
Also I think that having the function is not dependant on the default value at all is more readable so we also gain something. 

So I'll keep it as is for now. Thanks for the discussion :)

**Pull request merged**